src/main/java/bartworks/common/tileentities/multis/MTEDeepEarthHeatingPump.java

```diff
@@ -22,27 +22,33 @@
 import net.minecraft.entity.player.EntityPlayer;
 import net.minecraft.item.ItemStack;
 import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.util.StatCollector;
 import net.minecraftforge.common.util.ForgeDirection;
 import net.minecraftforge.fluids.Fluid;
 import net.minecraftforge.fluids.FluidRegistry;
 
+import com.gtnewhorizons.modularui.api.math.Pos2d;
+
 import gregtech.api.enums.GTValues;
 import gregtech.api.enums.ItemList;
 import gregtech.api.enums.Materials;
+import gregtech.api.gui.modularui.GTUITextures;
```

Comment:
```suggestion
import gregtech.api.enums.TierEU;
import gregtech.api.gui.modularui.GTUITextures;
```
Import so you can apply the change below in the UI